clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -190,7 +190,7 @@
                                         HEARTBEAT_INTERVAL_MS_DOC)
                                 .define(PARTITION_ASSIGNMENT_STRATEGY_CONFIG,
                                         Type.LIST,
```

Comment:
@granthenke  Type.CLASS refers to a single class, and Type.LIST at the moment takes only a list of Strings. Since multiple partitition assignor classes may be specified, it felt easier to handle the default value separately.